evalml/pipelines/pipeline_base.py

```diff
@@ -342,15 +320,14 @@ def parameters(self):
             component_parameters['pipeline'] = self._pipeline_params
         return component_parameters
 
-    @classproperty
-    def default_parameters(cls):
+    def default_parameters(self):
```

Comment:
I think you're planning on a separate PR to update the docs. In that PR I think we can make this a`property` along with `summary`, `problem_type`, `model_family`, `custom_name` etc so that they're documented!

Reply:
Decided to just make these attributes properties now, docs look like this: https://feature-labs-inc-evalml--2091.com.readthedocs.build/en/2091/generated/evalml.pipelines.PipelineBase.html#evalml.pipelines.PipelineBase 😁

Reply:
Very cool!

Reply:
Woot, thanks for the great suggestion! 🥳